rudder-core/src/test/scala/com/normation/rudder/services/policies/NodeConfigData.scala

```diff
@@ -481,16 +484,19 @@ class TestNodeConfiguration() {
 
   val abstractRoot = new File("/tmp/test-rudder-config-repo-" + DateTime.now.toString())
   abstractRoot.mkdirs()
+
   // config-repo will also be the git root, as a normal rudder
   val configurationRepositoryRoot = abstractRoot/"configuration-repository"
-
   //initialize config-repo content from our test/resources source
-  FileUtils.copyDirectory( new File("src/test/resources/configuration-repository") , configurationRepositoryRoot)
+
+  val location = getClass().getClassLoader().getResource("configuration-repository").getFile
```

Comment:
You should not change this line, this will only work in your dev environment, it should still be "src/test/resources/configuration-repository"

Reply:
I guess it changes everything done with it